using_openshift/output_deps.adoc

```diff
@@ -0,0 +1,53 @@
+= Dependency tree generation
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title: 
+
+toc::[]
+
+== Overview
+`build-chain` is an experimental command-line utility for OpenShift v3. It outputs the build dependency tree of a given image repository
+in JSON (default), DOT, or AST syntax. The user can specify an image repository name (and optionally a namespace and a tag) to build its
+dependency tree, call the tool without any arguments to build all dependency trees in the current namespace, or specify the `--all` flag to build all dependency trees in all available namespaces.
+ 
```

Comment:
This overview needs a little work, it should explain what  build dependency is.  eg:  This tool will start at the imagerepo:tag specified and determine what Builds will be triggered (due to ImageChangeTriggers on the BuildConfig) when this imagerepo:tag is updated.  Further, it analyzes the output of those Builds to determine if they in turn update an imagerepo:tag.  If they do, the tool continues to follow the dependency tree.  It then outputs the total graph of what imagerepo:tags will be impacted by an update to the top level imagerepo:tag.